src/include/miopen/graphapi/opgraph.h

```diff
@@ -0,0 +1,274 @@
+#pragma once
+
+#include <miopen/graphapi/graphapi_tensor.hpp>
+
+#include <algorithm>
+#include <vector>
+#include <unordered_map>
```

Comment:
```suggestion
#include <algorithm>
#include <unordered_map>
#include <vector>
```
Sort this list, please

Reply:
done